forte/processors/data_augment/algorithms/dictionary.py

```diff
@@ -0,0 +1,224 @@
+# Copyright 2020 The Forte Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import List
+import nltk
+from nltk.corpus import wordnet
+
+
+__all__ = [
+    "Dictionary",
+    "WordnetDictionary"
+]
+
+
+class Dictionary:
+    r"""
+    This class defines a dictionary for word replacement.
+    Given an input word and its pos_tag(optional), the dictionary
+    will outputs its synonyms, antonyms, hypernyns and hyponyms.
+    """
+    def get_synonyms(
+            self,
+            word: str,
+            pos_tag: str = "",
+            lang: str = "eng"
+    ) -> List[str]:
+        # pylint: disable=unused-argument
```

Comment:
If you put this pylint comment at a higher scope, it will be applied to the whole scope.